times_reader/main.py

```diff
@@ -292,11 +289,12 @@ def produce_times_tables(
                     f" - {', '.join(missing)}"
                 )
             else:
-                cols_to_drop = [x for x in df.columns if not x in mapping.xl_cols]
+                for (times_col, xl_col) in mapping.col_map.items():
+                    df[times_col] = df[xl_col]
```

Comment:
Perhaps a comment here explaining that this allows excel cols to be duplicated into multiple times columns?

Reply:
Done